build_dependencies.sh

```diff
@@ -33,22 +33,15 @@ usage()
   echo "  -h,--help               : Show this screen."
   echo "  --clean                 : Remove all build directories and clone implicit dependencies."
   echo "                          : (note that user-specified paths such as --itk-dir=<path> will not be deleted)."
-  echo "  --no-gui                : Do not build the ShapeWorks gui applications, which require Qt."
+  echo "  --no-gui                : Do not build the dependcies with GUI support, which require Qt."
   echo "                          : The GUI is built by default if qmake >= $QT_MIN_VER is found in the path."
-  echo "  --with-studio           : Build ShapeWorksStudio (default off)."
-  echo "  -b,--build-dir=<path>   : Build directory for ShapeWorks and its implicit dependencies (VXL, VTK, and ITK)."
-  echo "                          : By default uses a subdirectory of the current directory called 'build'."
-  echo "  -i,--install-dir=<path> : Install directory for ShapeWorks and its implicit dependencies (VXL, VTK, and ITK)."
-  echo "                          : By default uses a subdirectory of the current directory called 'install'."
+  echo "  -b,--build-dir=<path>   : Build directory for dependencies."
+  echo "                          : By default uses a subdirectory of the current directory called 'dependencies/build'."
+  echo "  -i,--install-dir=<path> : Install directory dependencies."
```

Comment:
Missing for: Install directory "for" dependencies

Reply:
👍